src/System Application/App/Cryptography Management/src/SignatureAlgorithm.Enum.al

```diff
@@ -44,4 +44,12 @@ enum 1446 SignatureAlgorithm implements "Signature Algorithm v2"
         Implementation = "Signature Algorithm v2" = "DSACryptoServiceProvider Impl.";
 #endif
     }
+    /// <summary>
+    /// Specifies the RSASSA-PSS algorithm implemented by RSA
+    /// </summary>
+    value(2; "RSASSA-PSS")
+    {
+        Implementation = SignatureAlgorithm = "RSA Impl.",
```

Comment:
Needs to be wrapped in `#if not CLEAN 24` pragmas, similar to above for `DSA`, `RSA`.